avm/res/sql/server/elastic-pool/main.bicep

```diff
@@ -5,7 +5,7 @@ metadata owner = 'Azure/module-maintainers'
 @description('Required. The name of the Elastic Pool.')
 param name string
 
-@description('Conditional. The name of the parent SQL Server. Required if the template is used in a standalone deployment.')
+@description('Required. The name of the parent SQL Server. Required if the template is used in a standalone deployment.')
```

Comment:
Please note, this parameter is only required if using this module in a standalone deployment. If you use it via its parent, the value would actually be optional, as the `server/main.bicep` module would pass its name through. Hence we specified those parameters (that is, parent resource names) as 'Conditional'. With UDT's we could actually change that - but I'd suggest to do this 'systematically' for all modules instead of just individual ones.